news/7170.feature.rst

```diff
@@ -0,0 +1,3 @@
+Implement a new hook utility function, :func:`collect_delvewheel_libs_directory`,
```

Comment:
It has to be the long form for Sphinx to hyperlink it properly unless there's a `.. currentmodule:: PyInstaller.utils.hooks` lurking nearby.

```suggestion
Implement a new hook utility function, :func:`~PyInstaller.utils.hooks.collect_delvewheel_libs_directory`,
```